charts/dependency-track/values.schema.json

```diff
@@ -59,6 +59,13 @@
     "apiServer": {
       "type": "object",
       "properties": {
+        "deploymentType": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": ["Deployment", "StatefulSet"]
+          }
+        },
```

Comment:
```suggestion
        "deploymentType": {
          "type": "string",
          "enum": ["Deployment", "StatefulSet"]
        },
```

Reply:
yeah that was dumb mistake 🙂

Reply:
fixed in [c1640fc](https://github.com/DependencyTrack/helm-charts/pull/102/commits/c1640fc73838b864ff2ed6e90aa443157aab51ca)